### PR TITLE
PRIME-2362 - Revised email content (2022-12)

### PR DESCRIPTION
**Pull request by neophyte57**
Mailhog on `localhost`:  http://localhost:8025/
Mailhog on `dev`:  https://mailhog-server-9c33a9-dev.apps.silver.devops.gov.bc.ca/

image

### CODE REVIEW

File: prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs [hunk 1]
```diff
@@ -8,30 +8,31 @@ namespace Prime.Configuration.Database
...
...et up their access, you must forward this notification and the information below to your <span class=\"underline\">whoever sets up PharmaNet user accounts. This is usually your PharmaNet software vendor but can also be someone like an IT department or authorized individual</span>. </strong><br><br><ol><li class=\"list-item-mb\"> Name of the community health practice: </li><li class=\"list-item-mb\"> Practice address: </li><li class=\"list-item-mb\"> PharmaNet site ID, if you have it:</li><li class=\"list-item-mb\"> If @Model.EnrolleeFullName accesses PharmaNet on behalf of another user, please ensure your PharmaNet software vendor knows who the other users are, and those users enrol in PRIME.<br><br></li></ol> Your software vendor can find @Model.EnrolleeFullName's details by following this link: <a href=\"@Model.TokenUrl\">@Model.TokenUrl</a> <strong>This link will expire in 10 days.</strong><br><br>Thank you,<br><br>PRIME Support <br>1-844-397-7463<br><a href=\"mailto:PRIMEsupport@gov.bc.ca\" target=\"_top\">PRIMEsupport@gov.bc.ca</a>", ModifiedDate = MultipleEmailContentUpdateNov30th2022, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = MultipleEmailContentUpdateNov30th2022},
```
Comment (bergomi02) [inline prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs:19]: Comparing to the content in JIRA ticket, there is an extra 'your' here.
image

**Comment by bergomi02** [comment]
A minor item. In renewal required email, there is an extra space. 
image
image

File: prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs (hunk 1 above)
Comment (neophyte57) [reply prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs:19]: Good catch! ... how did that happen 🤔

Comment (neophyte57) [reply prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs]: Updated template

Comment (bergomi02) [reply prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs]: confirmed that it is fixed.
image

**Comment by neophyte57** [comment]
> A minor item. In renewal required email, there is an extra space. image image

OK, fixed.

**Review by bergomi02 — APPROVED** [review]
LGTM!

File: prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs [hunk 2]
```diff
@@ -8,30 +8,31 @@ namespace Prime.Configuration.Database
     public class EmailTemplateConfiguration : SeededTable<EmailTemplate>
     {
```
Comment (neophyte57) [inline prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs:9]: Just a test:  Can I add comment to Merged PR?

File: prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs [hunk 3]
```diff
@@ -8,30 +8,31 @@ namespace Prime.Configuration.Database
...
         public static readonly DateTime EnrolleeRenewalRequiredUpdateAug17th2022 = new(2022, 8, 17, 8, 0, 0, DateTimeKind.Utc);
+        public static readonly DateTime MultipleEmailContentUpdateNov30th2022 = new(2022, 11, 30, 8, 0, 0, DateTimeKind.Utc);
```
Comment (neophyte57) [inline prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs:11]: Just a test: Can I add comment to Merged PR?

**Pull request merged**